radicle-cli/src/commands/node.rs

```diff
@@ -115,7 +118,7 @@ impl Args for Options {
 
         let op = match op.unwrap_or_default() {
             OperationName::Connect => Operation::Connect {
-                nid: nid.ok_or_else(|| anyhow!("a NID must be provided"))?,
+                nid: nid.ok_or_else(|| anyhow!("an NID must be provided"))?,
```

Comment:
Revert

Reply:
is it read as NEED or EN.I.D? if former `a` is correct one. I personally read it as EN.I.D, which requires `an`.